cliff.toml

```diff
@@ -8,12 +8,13 @@
 [changelog]
 # template for the changelog header
 header = """
-# Changelog\n
-All notable changes to this project will be documented in this file.\n
 """
 # template for the changelog body
 # https://keats.github.io/tera/docs/#introduction
 body = """
+Version {{ version | trim_start_matches(pat="v") }} of Clio, an XRP Ledger API server optimized for HTTP and WebSocket API calls, is now available.
+This release adds new features and bug fixes.
```

Comment:
For bugfix releases we use `This release adds bug fixes.`. But I'm not sure whether it is easy to implement a logic analysing version difference here.

Reply:
Ah, ok, I didn't notice this, I think I made it work